lib/client/client.wilson.js

```diff
@@ -27,6 +27,9 @@
   var _controllerProvider = null;
   var _provider           = null;
 
+  // Decorate Lodash with conveniences
```

Comment:
This seems to be reoccurring. It could make sense to make a new local module 'ht-lodash' that adds these conveniences and can be included in other scripts to simplify reuse.